version.sbt

```diff
@@ -1 +1 @@
-version := "2.5.2.0"
\ No newline at end of file
+version := "2.5.4.0"
```

Comment:
I'm assuming that the convention is that the first three numbers matches the akka version.